src/test/java/com/connectfour/DiscTest.java

```diff
@@ -0,0 +1,20 @@
+package com.connectfour;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DiscTest {
+
+    @Test
+    void testGetSymbol() {
+        Disc testDisc = new Disc("blue", 'X');
+        assertEquals('X', testDisc.getSymbol());
+    }
+
+    @Test
+    void testGetColor() {
+        Disc testDisc = new Disc("blue", 'X');
+        assertEquals("blue", testDisc.getColor());
+    }
```

Comment:
I don't think you need to test Disc.  It's just a data object.  It does not have any methods (besides getters).